src/Cake.Issues.DocFx/DocFxIssuesProvider.cs

```diff
@@ -36,38 +38,48 @@ protected override IEnumerable<IIssue> InternalReadIssues(IssueCommentFormat for
                 docRootPath = docRootPath.MakeAbsolute(this.Settings.RepositoryRoot);
             }
 
+            IEnumerable<LogEntryDataContract> logFileEntries = null;
+
+            var logFileContent = Encoding.UTF8.GetString(this.IssueProviderSettings.LogFileContent)
```

Comment:
You can use the [ToStringUsingEncoding extension method](https://cake-contrib.github.io/Cake.Issues.Website/api/Cake.Issues/ByteArrayExtensions/B203630E) with passing `true` for the `skipPreamble` argument.

Reply:
Oh, handy 😄